lib_comfyui/parallel_utils.py

```diff
@@ -3,6 +3,11 @@
 import multiprocessing.queues
 
 
+def clear_queue(queue):
+    while not queue.empty():
+        queue.get()
```

Comment:
should get with timeout here so we don't stall in case another thread yonked the value between the loop condition and here.

Reply:
I think we want a mutex here, no?

Reply:
might be a better solution, but that means the mutex should be used every time we want to get something

Reply:
we can't just mutex in the clear function?

Reply:
My concern is, what if another part of the code gets before the get here? This could invalidate some assumptions made at the call site of clear

Reply:
If we only mutex here, then it doesn't prevent other parts of the code to change the contents of the queue while we are emptying it

Reply:
yeah... I'll implement the get with timeout